sdk/python/tests/integration/registration/test_feature_store.py

```diff
@@ -11,68 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
```

Comment:
There's a lot being deleted here. Can you update the PR notes to explain why? I'm not fully familiar with the usage of all of these tests so I'd like to make sure I understand the implications of removing all of these tests (along with the other files).